tests/pageObjects/Main.js

```diff
@@ -0,0 +1,24 @@
+export class Main {
```

Comment:
`tests/pageObjects/Main.js` -> `tests/pages/Main.js` or `tests/objects/pages/Main.js` or `tests/objects/MainPage.js` 

However, the app has only one page, so perhaps the last option is the best.

Reply:
ok